web/src/main/java/com/epam/esm/web/advice/CustomErrorCode.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.esm.web.advice;
+
+public enum CustomErrorCode {
+    RECOURSE_EXIST("40901"),
+    RECOURSE_NOT_EXIST("40401"),
```

Comment:
Example: code should be `40401` if certificate not found and `40402` if tag

Reply:
done